dev/scss/_top-bar.scss

```diff
@@ -0,0 +1,11 @@
+.top-bar {
+  padding-top: 15px;
+  top: 0;
+}
+
+.logo {
```

Comment:
Seems like logo should be in a logo scss file unless it's somehow only used in topbar?